src/runner.ts

```diff
@@ -257,6 +259,22 @@ export const runAll = async (tests: Array<Test>, cwd: string, testSuite = 'autog
     jsonScoreLog.push(scoreLog)
   }
 
+  if (failed && all_or_nothing) {
+    summaryTable = summaryTable.map((row, index) => {
+      if (index == 0) {
+        return row
+      }
+      row[1] = '0'
+      return row
+    })
+
+    jsonScoreLog = jsonScoreLog.map(row => {
+      row.success = false
+      row.points = 0
+      return row
+    })
+  }

```

Comment:
I'm a little hesitant to replace all the entire table after it was created.   

Would a simpler impl would be the following in the lines above that construct the table in the first place (for both the success and fail cases)?
```
pointsString = test.points ? test.points.toString() : '0'
if (all_or_nothing) { pointsString = "—" }
...
summaryTable.push([test.name, pointsString, '✅'])
```

Reply:
This now appears in the main version of the code!